README.md

```diff
@@ -17,36 +17,36 @@
 [di]: https://docs.rs/specs-physics/badge.svg
 [dl]: https://docs.rs/specs-physics/
 
-**For when you want some [nphysics] in your [Specs]!**
-***Somewhat*** **better than sliced bread!**
+**Add [`nphysics`] to your [Specs] ECS entities!**
 
-Remember those "FooBarDefault" types in the [nphysics tutorial]?
-**specs-physics** provides [ECS Component]-based implementations for nphysics data sets,
+Remember those `FooBarDefault` types in the [`nphysics` tutorial]?
+**`specs-physics`** provides [ECS component]-based implementations for nphysics data sets,
 as well as faculties for synchronizing pose data to your position type of choice,
 and stepping functionality for stepping your simulations to a real good beat.
```

Comment:
Hmm, I think that most of the README should actually be pruned, and redirect people for more substantial documentation to the docs page, since most of this I think I ended up copy-pasting from `src/lib.rs`. Would you want to try at this?

Reply:
I had a look at it (several looks actually), but I'm afraid I'm not comfortable enough with neither amethyst nor specs-physics to do this. Sorry.

Reply:
No worries. I'll just merge this as-is and cut it down :) Thank you for all your help so far though!